[triton] Build error

[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/9157632/install-x64-windows-dbg-out.log)
**Host Environment**
- OS: win11 x64
- Compiler: revision

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install triton triton:x64-windows`

**Failure logs**
Installing 4/6 triton:x64-windows...
Building triton[core]:x64-windows...
-- Downloading https://github.com/JonathanSalwan/Triton/archive/243026c9c1e07a5ca834c4aaf628d1079f6a85ea.tar.gz -> JonathanSalwan-Triton-243026c9c1e07a5ca834c4aaf628d1079f6a85ea.tar.gz...
-- Extracting source E:/GitHub/vcpkg/downloads/JonathanSalwan-Triton-243026c9c1e07a5ca834c4aaf628d1079f6a85ea.tar.gz
-- Applying patch 001-fix-dependency-z3.patch
-- Applying patch 002-fix-capstone-5.patch
-- Applying patch 003-fix-python.patch
-- Using source at E:/GitHub/vcpkg/buildtrees/triton/src/079f6a85ea-4848cd5073.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:129 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j21
    Working Directory: E:/GitHub/vcpkg/buildtrees/triton/x64-windows-dbg
    See logs for more information:
      E:\GitHub\vcpkg\buildtrees\triton\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:111 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:41 (vcpkg_cmake_build)
  ports/triton/portfile.cmake:41 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)

 vcpkg-tool version: 2022-07-14-fb7e3b1bb3b1ca185fffe2be36b2fbc53198360f
    vcpkg-scripts version: 8ff168270 2022-07-19 (33 hours ago)

```
E:\GitHub\vcpkg\buildtrees\triton\src\079f6a85ea-4848cd5073.clean\src\libtriton\includes\triton/tritonTypes.hpp(17): fatal error C1083: Cannot open include file: 'boost/multiprecision/cpp_int.hpp': No such file or directory
```
Can you please remove `boost-uninstall boost-uninstall:x64-windows` then try again?

Thanks.

--- a/ports/triton/portfile.cmake
+++ b/ports/triton/portfile.cmake
@@ -15,14 +15,15 @@ string(COMPARE EQUAL "${VCPKG_CRT_LINKAGE}" "static" STATICCRT)
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
         "python" PYTHON_BINDINGS
+        "boost"  BOOST_INTERFACE
 )
 
-set(ADDITIONAL_OPTIONS )
+set(ADDITIONAL_OPTIONS "")
 if(PYTHON_BINDINGS)
     vcpkg_find_acquire_program(PYTHON3)
     list(APPEND ADDITIONAL_OPTIONS
-        -DPYTHON_EXECUTABLE=${PYTHON3}
-        )
+        "-DPYTHON_EXECUTABLE=${PYTHON3}"
+    )
 endif()
 
 vcpkg_cmake_configure(
@@ -46,8 +47,7 @@ vcpkg_fixup_pkgconfig()
 vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
 
 # Remove duplicate files
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include"
-                    "${CURRENT_PACKAGES_DIR}/debug/share")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include" "${CURRENT_PACKAGES_DIR}/debug/share")
 
 # Handle copyright
 file(INSTALL "${SOURCE_PATH}/LICENSE.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/triton/vcpkg.json
+++ b/ports/triton/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "version": "0.9",
-  "port-version": 3,
+  "port-version": 4,
   "description": "Triton is a Dynamic Binary Analysis (DBA) framework. It provides internal components like a Dynamic Symbolic Execution (DSE) engine, a dynamic taint engine, AST representations of the x86, x86-64, ARM32 and AArch64 Instructions Set Architecture (ISA), SMT simplification passes, an SMT solver interface and, the last but not least, Python bindings.",
   "homepage": "https://github.com/JonathanSalwan/Triton",
   "license": "Apache-2.0",
@@ -27,6 +27,13 @@
     "z3"
   ],
   "features": {
+    "boost": {
+      "description": "Use Boost as multiprecision library",
+      "dependencies": [
+        "boost-multiprecision",
+        "boost-numeric-conversion"
+      ]
+    },
     "python": {
       "description": "Build Triton with Python bindings",
       "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7238,7 +7238,7 @@
     },
     "triton": {
       "baseline": "0.9",
-      "port-version": 3
+      "port-version": 4
     },
     "trompeloeil": {
       "baseline": "41",

--- a/versions/t-/triton.json
+++ b/versions/t-/triton.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "6b86390804efe4afb7989e40a4cf84ad7877e85a",
+      "version": "0.9",
+      "port-version": 4
+    },
+    {
       "git-tree": "dd0d283c09ee98abf32a81d0833e81a9e5222028",
       "version": "0.9",
       "port-version": 3

